files/ko/learn/javascript/building_blocks/functions/index.html

```diff
@@ -284,12 +287,12 @@ <h3 id="실습_스코프랑_놀자">실습: 스코프랑 놀자</h3>
   output(x);
 }</pre>
  </li>
- <li>Save and reload again, and try this again in your JavaScript console:
+ <li>저장하고 다시 로드한 후 이것을 자바스크립트 콘솔에서 시도해 보세요.
   <pre class="brush: js notranslate">a();
 b();</pre>
  </li>
- <li>Both the <code>a()</code> and <code>b()</code> call should output the value of x — 1. These work fine because even though the <code>output()</code> calls are not in the same scope as <code>x</code> is defined in, <code>x</code> is a global variable so is available inside all code, everywhere.</li>
- <li>Finally, try updating your code like this:
+ <li>왜냐하면 비록 <code>output()</code> 호출이 <code>x</code>가 정의되어 있는 같은 스코프에 있지 않더라도, <code>x</code>는 전역 변수이므로 모든 코드 어디서든 이용 가능하기 때문에 이것들은 잘 작동합니다.</li>
```

Comment:
```suggestion
 <li>브라우저 뷰포트에 <code>a()</code> 와 <code>b()</code> 모두 x 값을 출력해야 합니다. 왜냐하면 비록 <code>output()</code> 호출이 <code>x</code>가 정의되어 있는 같은 스코프에 있지 않더라도, <code>x</code>는 전역 변수이므로 모든 코드 어디서든 이용 가능하기 때문에 이것들은 잘 작동합니다.</li>
```
Both the a() and b() call should print the value of x to the browser viewport. 
문장이 빠졌습니다.